openapi/sms/v1-review/docs/readme.md

```diff
@@ -0,0 +1,262 @@
+    | `apiKey` | `password` | API Key | N/A             |
+    | `region` | `select`   | Region  | `Asia`,`Europe` |
+- Features enabled:
+  - Whether your platform supports inbound messages
+- Contact details (we will contact you in case of an issue):
+  - Email
+
+And we subsequently provide:
+
+- Confirmation of setup completion on an environment (staging / production)
+- Emarsys SMS OAuth Client ID (to verify requests)
+
+### Authorization
+
+Requests sent by Emarsys will have an OAuth 2 token. This token should be validated by the Partner service. Emarsys will validate a Partner's requests using the provided (external) OAuth service and OAuth client ID
+
+| URI                                                       | Description                                                      |
+| --------------------------------------------------------- | ---------------------------------------------------------------- |
+| `https://smsemarsys.accounts.ondemand.com`              | SAP token issuer host                                            |
+| `https://smsemarsys.accounts.ondemand.com/oauth2/token` | SAP IAS access token endpoint                                    |
+| `https://smsemarsys.accounts.ondemand.com/oauth2/certs` | SAP IAS JSON Web Key Set (JWKS) certificate information endpoint |
+
+---
+
+## Workflows
+
+Below are the supported workflows of a partner service
+
+| Workflow          | Supported           | Required            |
+| ----------------- | ------------------- | ------------------- |
+| Client onboarding | <center>✅</center> | <center>✅</center> |
+| Outbound messages | <center>✅</center> | <center>✅</center> |
+| Inbound Messages  | <center>✅</center> | <center>❌</center> |
+
+### Client Configuration
+
+This process occurs during client onboarding of the Emarsys SMS channel. They input their required values in the custom fields that were provided during the [Partner Onboarding](#partner-onboarding)
+
+```mermaid
+sequenceDiagram
+  autonumber
+  participant Client as Client User
+  participant Emarsys as Emarsys SMS
+  participant Partner as Partner Service
+  Note right of Client: Onboarding
+  Client->>Emarsys: Client Configuration
+  Emarsys->>Partner: Client configuration
+  Note right of Partner: Partner stores the configuration used
+
+```
+
+1. Client user (marketer) will use the self-onboarding UI to setup their SMS channel. During this process they will select the partner and provide their values required in the custom fields.
+2. The SMS service will on-board the client, generate a unique client ID (to reference them) and [send the client configuration (the values entered in the custom fields) to the Partner service](../partner-api/partner-service.yaml/paths/~1clients~1{clientId}~1configuration/put).
+
+
+
+<!-- theme: warning -->
+> #### Attention!
+>
+> The Partner service must store the client configuration as it is it not stored within the SMS service.
+
+---
+
+#### Endpoints
```

Comment:
Maybe would be useful to mention Test connection endpoint in this section as well.

Reply:
nice catch! adding a section for this